src/PhpParser/Parser/InlineCodeParser.php

```diff
@@ -148,10 +145,7 @@ private function resolveConcatValue(Concat $concat): string
         }
 
         if ($concat->right instanceof String_ && str_starts_with($concat->right->value, '($')) {
-            $node = $this->betterNodeFinder->resolveNextNode($concat);
-            if ($node instanceof Variable) {
-                $concat->right->value .= '.';
-            }
+            $concat->right->value .= '.';
```

Comment:
on original solution, it using Token based check, I will update it.

Reply:
see original solution for it https://github.com/rectorphp/rector-src/pull/3864/files#diff-fdafd0c90f7922c2cc9a20cc36536ce4353aa04cb1c25ee4c08bf9f587934628

Reply:
Looks good. I wonder why no test is failing here though :thinking: 

Let's go with current version in this PR untill we find the breaking code, so we have it covered in test fixture for sure.

Reply:
yes, it definitelly breaking change, as we limit to variable on purpose to avoid next is func call, but let's see how large it break